src/lib.rs

```diff
@@ -641,14 +641,19 @@ impl Docker {
     /// constructs a new Docker instance for docker host listening at the given host url
     pub fn host(host: Url) -> Docker {
         match host.scheme() {
+            #[cfg(feature = "unix-socket")]
             "unix" => {
                 Docker {
                     transport: Transport::Unix {
-                        client: Client::with_connector(UnixSocketConnector),
+                        client: Client::with_connector(hyperlocal::UnixSocketConnector),
                         path: host.path().to_owned(),
                     },
                 }
             }
+
+            #[cfg(not(feature = "unix-socket"))]
+            "unix" => panic!("Unix socket support is disabled"),
```

Comment:
Can we add a note in the error message about how on could to enable it? The path to discovery is not obvious to those that are new to rust and cargo.

Reply:
i'm not sure that's necessary? It's on by default, so to see this error, one would have to first go and explicitly disable it.